quinn-proto/src/crypto/rustls.rs

```diff
@@ -411,8 +460,8 @@ impl crypto::PacketKey for PacketKey {
 /// This list prefers AES ciphers, which are hardware accelerated on most platforms.
 /// This list can be removed if the rustls dependency is updated to a new version
 /// which contains the linked change.
-static QUIC_CIPHER_SUITES: [&rustls::SupportedCipherSuite; 3] = [
-    &rustls::ciphersuite::TLS13_AES_256_GCM_SHA384,
-    &rustls::ciphersuite::TLS13_AES_128_GCM_SHA256,
-    &rustls::ciphersuite::TLS13_CHACHA20_POLY1305_SHA256,
+static QUIC_CIPHER_SUITES: [rustls::SupportedCipherSuite; 3] = [
```

Comment:
see comment above - can also be removed if you like (or being kept in order to keep quinn behavior independent of rustls defaults)